lib/tasks/clean.rake

```diff
@@ -37,6 +37,19 @@ namespace :clean do
     cleaner.deactivate_invalid_subscriptions(dry_run: dry_run)
   end
 
+  desc "Remove SubscriberLists with no subscriptions"
+  task remove_empty_subscriberlists: :environment do
+    dry_run = is_dry_run?
+    count = 0
+    SubscriberList.find_each do |l|
+      if l.subscriptions.count.zero?
+        count += 1
+        l.destroy unless dry_run
```

Comment:
I wonder if this could be added as a class in the `Clean` module, so we could add tests for it?